profiles/cli.py

```diff
@@ -50,3 +52,73 @@ def SetOrcidWebhooksCommand(profiles: Profiles, orcid: Dict[str, str], orcid_cli
         uri = url_for('webhook._update', payload=uri_signer.dumps(profile.orcid),
                       _external=True)
         orcid_client.set_webhook(profile.orcid, uri, access_token)
+
+def foo(profiles: Profiles, orcid: Dict[str, str], orcid_client: OrcidClient,
+                            uri_signer: URLSafeSerializer) -> None:
+    access_token = orcid.get('webhook_access_token')
+
+    problem_profiles = {}
+
+    bar = {
+    "le6iu26f": "409 Client Error: Conflict for url: https://api.orcid.org/v2.1/0000-0002-2301-6721/record",
+    "z0hvtc2g": "409 Client Error: Conflict for url: https://api.orcid.org/v2.1/0000-0003-3363-6702/record",
+    "5cw1n9xf": "404 Client Error: Not Found for url: https://api.orcid.org/v2.1/None/record",
+    "4qku68vz": "404 Client Error: Not Found for url: https://api.orcid.org/v2.1/None/record",
+    "ssa5q9zp": "409 Client Error: Conflict for url: https://api.orcid.org/v2.1/0009-0008-6904-346X/record",
+    "xv82wdz4": "404 Client Error: Not Found for url: https://api.orcid.org/v2.1/None/record",
+    "7ba0avjv": "404 Client Error: Not Found for url: https://api.orcid.org/v2.1/None/record",
+    "yc376ncs": "404 Client Error: Not Found for url: https://api.orcid.org/v2.1/None/record",
+    "eux3p6tw": "404 Client Error: Not Found for url: https://api.orcid.org/v2.1/None/record",
+    "8n60qsjh": "404 Client Error: Not Found for url: https://api.orcid.org/v2.1/None/record"
+    }
+
+
+    try:
+        for profile in profiles.list():
```

Comment:
most of the time we'll only want to query profiles created in a given time period (days? months?)